### DISCUSSION TITLE
DediConnect - No desktop experiance

### DISCUSSION THREAD

**Discussion by scarecr0w12** [gameserverapp/Platform#543] [Suggestions]
Having to use the desktop experience for windows server is just weird to have as a requirement. Once GSA is installed, there is almost no reason to access the server desktop, and anything you'd need to do, can be done via powershell or similar utilities. Requiring desktop adds bloat such as RAM that is used for an unneeded desktop, Storage usage for unneeded files and similar for the desktop, and increased CPU usage due to desktop related processes.

There is no reason whatsoever to need or have a desktop for a Box that is acting as a server. Please give us the option to headless install on Windows based machines.

**Comment by mvaessen** [comment]
When issues come up, where people need to do things manually, they wont be able to work with Powershell.
I don't see the benefit of making this harder to manage for folks
